core/src/test/scala/integration/kafka/server/QuorumTestHarness.scala

```diff
@@ -444,7 +444,6 @@ object QuorumTestHarness {
 
   // The following parameter groups are to *temporarily* avoid bugs with the CONSUMER group protocol Consumer
   // implementation that would otherwise cause tests to fail.
-  def getTestQuorumAndGroupProtocolParametersClassicGroupProtocolOnly_KAFKA_17960: stream.Stream[Arguments] = getTestQuorumAndGroupProtocolParametersClassicGroupProtocolOnly
   def getTestQuorumAndGroupProtocolParametersClassicGroupProtocolOnly_KAFKA_17961: stream.Stream[Arguments] = getTestQuorumAndGroupProtocolParametersClassicGroupProtocolOnly
```

Comment:
This can also be removed. And the next one. Only the final of these 4 definitions is used after this change.

Reply:
Oh, so only `getTestQuorumAndGroupProtocolParametersClassicGroupProtocolOnly_KAFKA_18034` is still used.

Reply:
I just removed the unused ones.